blogposts/inside-rust/01-announcement.md

```diff
@@ -128,31 +130,43 @@ Throughout, the  unwind generated by `panic!` will be referred to as
   * With `panic=unwind`: all types of unwinding behave normally
   * With `panic=abort`: all types of unwinding abort the program
 
+This proposal provides 2 ABIs, each suited for different purposes: you would
+generally use `extern "C"` when interacting with C APIs (making sure to avoid
+destructors where `longjmp` might be used), and `extern "C unwind"`
+when interacting with C++ APIs. The main advantage of this proposal is that
+switching between `panic=unwind` and `panic=abort` can never introduce UB, but
+may cause your program to abort instead of unwind.
+
 ### Proposal 2: Introduce `"C unwind"`, forced unwinding always permitted
 
 This is the same as the previous design, except that when compiled with
 `panic=abort`, forced unwinding would *not* be intercepted at `"C unwind"` ABI
-boundaries; that is, they would behave normally, without causing the program to
-abort. `panic`-unwind and non-forced foreign exceptions would still cause the
-program to abort.
+boundaries; that is, they would behave normally (though still UB if there are
+any destructors), without causing the program to abort. `panic`-unwind and
+non-forced foreign exceptions would still cause the program to abort.
 
-### Proposal 3:  No new ABI
+### Proposal 3: No new ABI
 
 * `panic=unwind`: unwind behaves normally
 * `panic=abort`:
   * `panic`-unwind: does not exist; `panic!` aborts the program
   * foreign unwinding, no destructors: unwind behaves normally
   * foreign unwinding with destructors: undefined behavior
 
+The main advantage of this proposal is its simplicity: there is only one ABI and
+the behavior of `panic=unwind` is identical to that of `-fno-exceptions` in C++.
+However this comes with the downside that switching to `panic=abort` may in some
+cases introduce UB (though only in unsafe code) if FFI calls unwind through Rust
+code.
+
 ## Comparison table for the proposed designs
 
 In this table, "UB" stands for "undefined behavior". Some instances of UB can
 be detected at runtime, but the code to do so would impose an undesirable
 code-size penalty; the group recommends generating such code only for debug
 builds. These cases are marked "UB (debug: abort)"
 
-Note that forced unwinding always causes undefined behavior when it traverses
-stack frames with destructors.
+Note that unwinding through a frame without running its destructors (e.g. because they have been optimized out by `panic=abort`) is always undefined behavior, except if the frame has no destructors.
```

Comment:
```suggestion
Note that unwinding through a frame that has destructors without running those destructors (e.g. because they have been optimized out by `panic=abort`) is always undefined behavior.
```